cluster/juju/layers/kubernetes-master/config.yaml

```diff
@@ -30,3 +30,7 @@ options:
     type: string
     default: "RBAC"
     description: kube-apiserver --authorization-mode <value>
+  cluster-context:
+    type: string
+    default: "juju-context"
```

Comment:
Agreed we should have the juju_model_name by default there. AFAIK there is no way to grab the model name from within a charm. What we could do is to have the conjure-up spell fill in the cluster-context config param. Would that be ok?